### DISCUSSION TITLE
Long figure caption

### DISCUSSION THREAD

**Discussion by fcharih** [quarto-dev/quarto-cli#5524] [Q&A] [answered]
Hi. I apologize in advance if this has been answered before, but I didn't find an answer myself.

Is there any way I can prepare long figure captions/descriptions in Quarto. In LaTeX, you can define a short and long caption `\caption[Short caption]{Long caption}`. I am looking for a way to do something as follows. 👇🏾

[FIGURE HERE]

**Figure 1. This is the figure title that appears in the list of figures.**
This is a long description of the figure that cannot be separated from the figure title, but that does not appear in the list of figures.

**Comment by baptiste** [comment, marked as answer]
`fig-scap` lets you define that `[Short caption]`

**Reply by fcharih** [reply]
Awesome! Thank you!

**Comment by mcanouil** [comment]
FYI, related to #4565 in some aspects.

**Reply by fcharih** [reply]
Thank you!
